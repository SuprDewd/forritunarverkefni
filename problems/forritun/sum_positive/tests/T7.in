-807
-443
854
534
-931
118
-800
763
-814
-17
