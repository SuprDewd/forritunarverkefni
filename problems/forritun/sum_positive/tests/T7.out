2269
4
6
